package.json

```diff
@@ -16,6 +16,7 @@
     "test": "lerna run test --since origin/master -- --coverage",
     "test:all": "lerna run test -- --coverage",
     "lint": "lerna run lint --since origin/master --",
+    "lint:docs": "git ls-files | grep -h \".md\" | xargs vale",
```

Comment:
Unfortunately this will likely  not work across all OS's (e.g. Windows). I wonder if there is an cross-env npm package that could help here.

Reply:
Thanks for your suggestion. I will try wrapping this command in [shelljs](https://www.npmjs.com/package/shelljs) and then add to the lint-staged.